app/src/main/java/ar/scacchipa/jsonplaceholderviewer/domain/MiddleCommentsUserCase.kt

```diff
@@ -0,0 +1,17 @@
+package ar.scacchipa.jsonplaceholderviewer.domain
```

Comment:
maybe we can, in the future not necesary now, to have two kind of folders inside data folder, like one for repositories and other one for data sources